README.md

```diff
@@ -1,59 +1,158 @@
+2. To confirm the instance is up, run `pipenv run tim -u localhost ping` or visit http://localhost:9200/. This should produce a log that looks like the following:
+    ```
+    2024-02-08 13:22:16,826 INFO tim.cli.main(): OpenSearch client configured for endpoint 'localhost'
 
-```bash
-docker pull opensearchproject/opensearch:latest
-docker pull opensearchproject/opensearch-dashboards:latest
-docker compose up
-```
+    Name: docker-cluster
+    UUID: RVCmwQ_LQEuh1GrtwGnRMw
+    OpenSearch version: 2.11.1
+    Lucene version: 9.7.0
+
+    2024-02-08 13:22:16,930 INFO tim.cli.log_process_time(): Total time to complete process: 0:00:00.105506
+    ```
+
+### Running Opensearch and OpenSearch Dashboards locally with Docker
+
+You can use the included Docker Compose file ([compose.yaml](compose.yaml)) to start an OpenSearch instance along with OpenSearch Dashboards, "[the user interface that lets you visualize your Opensearch data and run and scale your OpenSearch clusters](https://opensearch.org/docs/latest/dashboards/)". Two tools that are useful for exploring indices are [DevTools](https://opensearch.org/docs/latest/dashboards/dev-tools/index-dev/) and [Discover](https://opensearch.org/docs/latest/dashboards/discover/index-discover/).
+
+**Note:** To use Discover, you'll need to create an index pattern. When creating the index pattern, decline the option to set a date field. When set, it detects a date field in our indices but then crashes trying to use it. When prompted, enter an index or alias to pull patterns from, and it will automatically be configured to work well enough for initial data exploration.
 
-To confirm the instance is up, run `pipenv run tim -u localhost ping`.
+1. Run the following command:
+    ```bash
+    docker pull opensearchproject/opensearch:latest
+    docker pull opensearchproject/opensearch-dashboards:latest
+    docker compose up
+    ```
 
-To access the Dashboard, access <http://localhost:5601>.
+2. To confirm the instance is up, run `pipenv run tim -u localhost ping` or visit http://localhost:9200/.
 
-DevTools is useful for writing/testing OpenSearch queries.
+3. Access OpenSearch Dashboards through <http://localhost:5601>.
 
-Discover is useful for browsing data. An index pattern will be required to use this tool. Note: do not set a date filed (choose the option to skip selecting a date field). It detects a date field in our indexes but then crashes trying to use it. Once you skip the data select field, just enter an index or alias to pull patterns from and it will automatically be configured to work well enough for initial data exploration.
+For a more detailed example with test data, please refer to the Confluence document: [How to run and query OpenSearch locally](https://mitlibraries.atlassian.net/wiki/spaces/D/pages/3586129972/How+to+run+and+query+OpenSearch+locally).
 
-### OpenSearch on AWS
+### Indexing a record into OpenSearch locally with Docker
```

Comment:
I might suggest rewording to "Index sample records into local Opensearch Docker container".

Two reasons:

1. It's not a single record, but a batch of records in a single file
2. The code provided is running the native python code, but it will index _into_ a local Opensearch container.

Reply:
I updated the heading to read "Index records into local OpenSearch Docker container". Left out the world "sample" to keep it generalizable 🤓 .